api/loaders/postgres_loader.py

```
@@ -1,32 +1,41 @@
+
+
 class PostgresLoader(BaseLoader):
     """
     Loader for PostgreSQL databases that connects and extracts schema information.
     """
 
-    # DDL operations that modify database schema
+    # DDL operations that modify database schema  # pylint: disable=duplicate-code
     SCHEMA_MODIFYING_OPERATIONS = {
         'CREATE', 'ALTER', 'DROP', 'RENAME', 'TRUNCATE'
     }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)